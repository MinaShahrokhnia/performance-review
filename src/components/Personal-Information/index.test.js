import React from 'react';
import { shallow } from 'enzyme';
import PersonalInformation from '.'

const informationPersonal = shallow(<PersonalInformation/>);

it('renders `personal-information` component', () => {
    expect(informationPersonal).toMatchSnapshot();
})