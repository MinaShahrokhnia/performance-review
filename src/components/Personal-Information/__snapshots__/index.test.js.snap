// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders \`personal-information\` component 1`] = `ShallowWrapper {}`;
